core/detekt-baseline.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" ?>
+<SmellBaseline>
+  <ManuallySuppressedIssues/>
+  <CurrentIssues>
+    <ID>ComplexCondition:ScrollArea.kt$&lt;no name provided&gt;$isOverscrollTop || isOverscrollBottom || isOverscrollLeft || isOverscrollRight</ID>
```

Comment:
Do we need to keep track the issues? Seems like a pain to maintain.

Is it possible to just set a threshold on the number of allowed warnings instead?

Reply:
These issues are baseline, they will be automatically generated when you fire `gradlew detektBaseline`. You can build a baseline again after cleaning up after the first time. No, they are not mandatory to push to GIT; it is up to you. The reports will ignore the issues that are already present in the baseline.

Reply:
> there is a threshold number yes.

Let's use that then. Figure out the current number of issues detekt picks up and use that as the threshold. I am not interested in allowing extra issues in future PRs. Future PRs with more issues need to fail the checks instead.